src/Xamarin.Android.Build.Tasks/Xamarin.Android.SkipCases.projitems

```diff
@@ -16,167 +16,167 @@ Copyright (C) 2018 Xamarin. All rights reserved.
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Loader" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.LocalBroadcastManager" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Media.Compat" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Percent" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Print" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Recommendation" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Recommendation" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.RecyclerView.Selection" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Slices.Builders" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Slices.Core" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Slices.View" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.SlidingPaneLayout" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.SwipeRefreshLayout" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Transition" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.TV.Provider" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v13" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v14.Preference" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v17.Leanback" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v17.Preference.Leanback" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v4" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v7.AppCompat" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v7.CardView" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v7.GridLayout" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v7.MediaRouter" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v7.Palette" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v7.Preference" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v7.RecyclerView" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.v8.RenderScript" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Vector.Drawable" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.VersionedParcelable" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.ViewPager" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Wear" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.Wearable" />
-    <_AndroidAssemblySkipCases Include="Xamarin.Android.Support.WebKit" />
+    <_AndroidAssemblySkipCases Include="Xamarin.Android.Arch.Core.Common" ><SkipAndroidResourceProcessing>True</SkipAndroidResourceProcessing></_AndroidAssemblySkipCases>
```

Comment:
Do we really want these all on line line?

We could probably instead use an [`ItemGroupDefinition`](https://docs.microsoft.com/en-us/visualstudio/msbuild/itemdefinitiongroup-element-msbuild?view=vs-2017):

```xml
<ItemGroupDefinition>
  <_AndroidAssemblySkipCases>
    <SkipAndroidResourceProcessing>True</SkipAndroidResourceProcessing>
  </_AndroidAssemblySkipCases>
</ItemGroupDefinition>
```

This would avoid needing to modify all these lines entirely.

Reply:
not we dont. This was just a quick hack. and ItemGroupDefinition will be MUCH better.